wsagent/che-core-api-debug-shared/src/main/java/org/eclipse/che/api/debug/shared/model/impl/FieldImpl.java

```diff
@@ -10,49 +10,51 @@
  */
 package org.eclipse.che.api.debug.shared.model.impl;
 
-import java.util.List;
-import org.eclipse.che.api.debug.shared.model.Variable;
+import org.eclipse.che.api.debug.shared.model.Field;
+import org.eclipse.che.api.debug.shared.model.SimpleValue;
 import org.eclipse.che.api.debug.shared.model.VariablePath;
 
 /** @author Anatoliy Bazko */
-public class FieldImpl extends VariableImpl {
+public class FieldImpl extends VariableImpl implements Field {
   private boolean isFinal;
   private boolean isStatic;
   private boolean isTransient;
   private boolean isVolatile;
 
   public FieldImpl(
       String name,
-      boolean isExistInformation,
-      String value,
+      SimpleValue value,
       String type,
       boolean isPrimitive,
-      List<Variable> variables,
       VariablePath variablePath,
       boolean isFinal,
       boolean isStatic,
       boolean isTransient,
       boolean isVolatile) {
-    super(type, name, value, isPrimitive, variablePath, variables, isExistInformation);
+    super(type, name, value, isPrimitive, variablePath);
     this.isFinal = isFinal;
     this.isStatic = isStatic;
     this.isTransient = isTransient;
     this.isVolatile = isVolatile;
   }
 
-  public boolean isFinal() {
+  @Override
+  public boolean isIsFinal() {
```

Comment:
Probably because this is a getter and the variable starts with `is` prefix.
Anyway looks weird.